### ISSUE TITLE
UI/UX changes

### ISSUE THREAD

**Issue by Github-com-foss** [92lleo/WhatsappWebToGo#48]
Pls provide Original Whatsapp app like UI & UX changes in whatsapp web.

**Comment by 92lleo** [comment]
As mentioned, the app just mirrors web.whatsapp.com. There would be a lot of work involved in recreating the whatsapp ui and reading/setting everything. I don't have the time/intention to do that, but I'm open to input, but until now it's not planned

**Comment by ronz91** [comment]
Hi there 

Thanks for this great app.

My idea for a solution to this would be a button that basically alternates betweens left and right hand sides of the UI and zooms/scales to a level that the user can set in settings.

So basically when viewing on the left where the contact list is, clicking the button would move to the right where the chat window is and zooming/scaling to the user set view.

**Comment by 92lleo** [comment]
Yes, I guess this is the easiest way. with #52 the beginning of this is already done. Same would have to be done for the chat list and initial screen with qr code, scrolling could then be done via swipeing or buttons
